Program.cs

```diff
@@ -6,17 +6,36 @@
 
 namespace CurrentValueCalculator_HW2
 {
-    enum TransactionType
-    {
-        Buy,
-        Sell
+    enum TransactionType {
+        Buy, Sell
     }
+    class Program {
+        static void Main(string[] args) {
+            Console.WriteLine("Input the nominal of the trade:");
+            string userInput = Console.ReadLine();
+            int nominal = int.Parse(userInput ?? "0");
 
-    class Program
-    {
-        static void Main(string[] args)
-        {
+            Console.WriteLine("Input the trade price of the deal:");
+            userInput = Console.ReadLine();
+            double tradePrice = double.Parse(userInput ?? "0");
 
+            Console.WriteLine("Input the transaction type:");
+            userInput = Console.ReadLine();
+            TransactionType trcType = (TransactionType)Enum.Parse(typeof(TransactionType), userInput ?? "0", true);
```

Comment:
this condition doesn't make sense here - `userInput ?? "0"`
default value "0" won't be parsed to TransactionType anyway